etc/pylintrc

```diff
@@ -1,6 +1,7 @@
 [MASTER]
 extension-pkg-whitelist=numpy,scipy
+disable=C0103
 
```

Comment:
I didn't particularly intend to include this in this PR. It was more generally across the IMPROVER repository, we have used lots of names that fall foul of this. The Pylint score jumps to 9.5 is we exclude the naming provisions of pylint. It has not been agreed that this change should go in so I will remove it from this PR. Thanks for spotting it.